starport-core/src/main/scala/com/krux/starport/db/tool/CreateUsers.scala

```diff
@@ -0,0 +1,63 @@
+package com.krux.starport.db.tool
+
+import com.krux.starport.config.StarportSettings
+import com.krux.starport.Logging
+import com.krux.starport.db.WaitForIt
+import com.krux.starport.db.table._
+import com.typesafe.config.ConfigValueFactory
+import scopt.OptionParser
+import slick.jdbc.PostgresProfile.api._
+import slick.jdbc.PostgresProfile.backend.DatabaseDef
```

Comment:
let's have the import order changed to be consistent with other files

```
java.*
--new line--
scala.*
--new line--
other imports
--new line--
com.krux.*
```

Reply:
done.